src/Strings.cpp

```diff
@@ -0,0 +1,157 @@
+        "When I was younger, I always wanted to become like Draco the _ Demon. Legend has it, he single handedly caused the Chernobyl meltdown.",
+        "I idolize my tough-skinned _ Grandpa. He fought in the legendary Demons and Angels War and obliterated an entire battalion of cherubs.",
+        "My favorite character in my TV show just died. He was a _ demon with the perfect balance of cynicism and ruthlessness.",
+        "When I was in college, my professor, Dr. _ Demon, inspired me to pursure a double major in Sadism and Witchcraft. Absolutely no regrets.",
+        "My best friend was always there for me. If it wasn't for that amazing _ demon, Hell would be the absolute worst.",
+        "If it wasn't for Yolanda the _ Torturer's influence on my life on Earth, I might have ended up in Heaven. That would have been absolute torture!"} });
+
+    tierList.push_back("Hopeless Mortal");
+    tierList.push_back("Permanent Resident of the Friend Zone (aka Hell)");
+    tierList.push_back("Average Douche");
+    tierList.push_back("Satan�s Wingman");
+    tierList.push_back("King of the Underworld");
+
+    keys = {"A","B","C","D","E","F","G","H","I","J","K",
+        "L","M","N","O",
+        "P","Q","R","S",
+        "T","U","V","W",
+        "X","Y","Z","Num0",
+        "Num1","Num2","Num3","Num4",
+        "Num5","Num6","Num7","Num8",
+        "Num9","Escape","LControl","LShift",
+        "LAlt","LSystem","RControl","RShift",
+        "RAlt","RSystem","Menu","LBracket",
+        "RBracket","Semicolon","Comma","Period",
+        "Quote","Slash","Backslash","Tilde",
+        "Equal","Hyphen","Space","Enter",
+        "Backspace","Tab","PageUp","PageDown",
+        "End","Home","Insert","Delete",
+        "Add","Subtract","Multiply","Divide",
+        "Left","Right","Up","Down",
+        "Numpad0","Numpad1","Numpad2","Numpad3",
+        "Numpad4","Numpad5","Numpad6","Numpad7",
+        "Numpad8","Numpad9","F1","F2",
+        "F3","F4","F5","F6",
+        "F7","F8","F9","F10",
+        "F11","F12","F13","F14",
+        "F15","Pause","KeyCount","Hyphen",
+        "Backspace","Backslash","Semicolon","Enter" };
+}
+
+std::string Strings::getString(std::string key) {
+    return strings[key];
+}
+
+std::string Strings::getResponse(std::string key) {
+    std::string response = (responses[key])[(std::rand() % 10)];
+    for (int x = 49; x < response.size(); x += 52) {
+        response.insert(x, "\n");
+    }
+    return response;
+}
+
+std::string Strings::getTier(int rank) {
+    return tierList[rank];
+}
+
+std::string Strings::getPreference(std::string key, int color) {
+    std::string colText;
+    switch (color) {
+        case 0: colText = "red"; break;
+        case 1: colText = "blue"; break;
+        case 2: colText = "green"; break;
+    }
+    std::string pref;
+    pref = preferences[key][(std::rand() % 10)];
+    int index = pref.find("_");
```

Comment:
Replace the "_" with the color so it doesn't show up in the dialogue box. Also, please change it to something more unique such as "***" instead of just an underscore